src/20240729_AaveV3Ethereum_JulyFundingUpdate/AaveV3Ethereum_JulyFundingUpdate_20240729.sol

```diff
@@ -47,7 +48,7 @@ contract AaveV3Ethereum_JulyFundingUpdate_20240729 is IProposalGenericExecutor {
     AaveV3Ethereum.COLLECTOR.transfer(
       AaveV3EthereumAssets.USDe_A_TOKEN,
       address(this),
-      IERC20(AaveV3EthereumAssets.USDe_A_TOKEN).balanceOf(COLLECTOR) - 1e18
+      IScaledBalanceToken(AaveV3EthereumAssets.USDe_A_TOKEN).scaledBalanceOf(COLLECTOR) - 1e18
```

Comment:
@defijesus  apart from leaving the collector with 1e18 + interest, is there any other point in using scaled balance instead of balanceOf?